playsound3/playsound3.py

```diff
@@ -194,7 +204,9 @@ def _playsound_gst_legacy(sound: str) -> None:
     logger.debug("gstreamer: finishing play %s", sound)
 
 
-def _send_winmm_mci_command(command):
+def _send_winmm_mci_command(command: str) -> Any:
+    if sys.platform != "win32":
+        raise RuntimeError("WinMM is only available on Windows systems.")
```

Comment:
Let's remove this check.

In other backends we don't check whether the backend is compatible with user's system. When a user selects this backend manually (instead of using the default), it's their responsibility to make sure it runs.